scripts/easy-spinnaker/readme.md

```diff
@@ -77,7 +77,7 @@ c\) The application URL is
 [[http://10.168.3.10]{.underline}](http://10.168.3.10/):{DECK NodePort}
 can be obtained with this command on the master node:
 
-kubectl get svc spin-deck-ui -n oes -o
+kubectl get svc spin-deck-np -n oes -o
```

Comment:
The latest OES creates spin-deck-ui. Why is this being changed back to spin-deck-np?

Reply:
Fixed

Reply:
Doesn't seem it is fixed.

Reply:
Now it is.